docs/source/configuration/environmentvariables.md

```diff
@@ -24,13 +26,39 @@ You could, for example, build your Volto application, then start it in productio
 pnpm build && RAZZLE_API_PATH=https://plone.org pnpm start:prod
 ```
 
+```{glossary}
+:sorted:
+`HOST`
+    Represents the host or IP address on which a server should listen.
+    Default is `0.0.0.0`.
+
+`PORT`
+    Used to specify the port on which a web server should listen for incoming requests.
+    Default is `3000`.
+
+`RAZZLE_API_PATH`
+    Used to configure the API path for the application.
```

Comment:
People try to use this one too much, so we should be clear that it's not recommended.

```suggestion
    Used to configure the API path for the application.
    Setting this is not usually necessary, because the Volto client will access the API at the same location where the frontend is served, and proxy the requests to the backend API.
    You can set this to make the Volto client connect directly to the API on a different host, but this can be complicated because of cross-origin restrictions.
```